### PR TITLE
Documentation/scripts update

### PR DESCRIPTION
**Pull request by pavankum**
README updates to qm benchmarks directory with file manifest and explaining the order in which to run the scripts.

### CODE REVIEW

**Comment by pavankum** [comment]
@jthorton I pushed one more update (waiting for the commit to show up here), I think it's ready for review after that.

File: inputs-and-results/benchmarks/qm-benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+   ```
+   b. Concatenate the metrics files in csv format generated above 
+   ```shell
+   python 03-b-join-metrics.py 03-outputs/* -o 03-metrics.csv
+   ```
+
+4) Plot the metrics by running the `04-plot-metrics.py` script, which will be stored in `04-outputs`. The directory `04-outputs/full` contains the metrics on overall set and the other directories are subsets of R-groups.
+
+# File Manifest
+
```
Comment (jthorton) [inline inputs-and-results/benchmarks/qm-benchmarks/README.md:56]: The descriptions are great can we reformat this slightly to make it clear what is a script and what is the associated output is? Maybe something like
-01-setup.py - Description ...
    - list of outputs - descriptions

**Review by jthorton — COMMENTED** [review]
Thanks @pavankum this looks great, just one small change to the qm-benchmarks readme and it is good to go. Should we add a notebook for that step as well do you think?

File: inputs-and-results/benchmarks/qm-benchmarks/README.md (hunk 1 above)
Comment (pavankum) [reply inputs-and-results/benchmarks/qm-benchmarks/README.md:60]: I took a shot at it, please take a look when you have time.

**Comment by pavankum** [comment]
> Should we add a notebook for that step as well do you think?

I am not sure, it would be too much clutter in a single notebook, but I can make one.

**Comment by pavankum** [comment]
On second thoughts, I think the benchmark scripts are self-explanatory and we added a lot to the README section, it should be fine.

**Review by jthorton — APPROVED** [review]
Thanks @pavankum this is great, ill take a go at fixing the conflicts since I caused them ...

**Pull request merged**